[49] Fix for caret jumping backwards

wirk opened the pull request.

wirk committed:
[49] Fix for caret jumping backwards

wirk committed:
Merge pull request #1 from vuejs-tips/master

update head

LeeW32 commented:
Hi @wirk Just trying to build this project to be able to incorporate your fix - but am new to the JavaScript world in general!  Getting in such a mess trying to build this on a windows machine. Have cloned project, run "npm install", then installed vue globally, then changed the 'cp' in the build command to 'copy' (guess this wasn't created in windows originally). Now Vue is telling me to use 'poi' so not sure how to progress?  Any ideas?

wirk committed:
Merge branch 'master' into bugfix/49-caret-jumps-backwards

wirk committed:
Fix build errors

wirk committed:
update prebuilt modules

wirk commented:
Hi @LeeW32 I have merged back the current master into my branch (that should fix the error you were getting). Also:
* fixed two small build warnings (whitespace, duplicate variable declaration)
* updated the build command to point directly to the index.js as the entry file rather than the src directory, which resulted in the build failing
* updated the prebuilt modules

LeeW32 commented:
Hi @wirk - that's brilliant - thanks - working well (just had to replace the cp command with "copy dist\\VueTheMask.umd.min.js dist\\vue-the-mask.js")

@neves any plans for this library?

victorwpbastos commented:
Please merge this PR. It resolves so many bugs.

kaskar2008 commented:
We have created a fork and published a package with the reference to the author of this project. Also we have merged some major PRs into our fork. If you need more features - feel free to contribute, we are open to PR :)

[Package](https://www.npmjs.com/package/ke-the-mask)
[Repo](https://github.com/KazanExpress/ke-the-mask)
